### PR TITLE
Update profile pic and background

### LINKED ISSUE #276: [#266-Re-design of Shared Wallets Creation process] - Add background, profile picture

### PR DESCRIPTION
**Pull request by mikozet**

### CODE REVIEW

File: src/views/OnboardingOrganization.js [hunk 1]
```diff
@@ -195,38 +225,61 @@ const OrganizationStepEmail = ({ values, onDisabledChange, onChange }) => {
...
-      <Typography align="center" gutterBottom variant="h2">
+    <Box className={classes.organizationStepAvatarContainer}>
+      <Typography align="center" gutterBottom variant="h6">
         {translate('OnboardingOrganization.headingAvatar')}
       </Typography>
-      <Typography>{translate('OnboardingOrganization.bodyAvatar')}</Typography>
-      <Box mt={4}>
+      <Box mb={4} mt={4}>
         <AvatarUploader
+          shouldHaveIndicator
```
Comment (louilinn) [inline src/views/OnboardingOrganization.js:254]: Does this mean it is set to true?

**Review by louilinn — COMMENTED** [review]
I think code looks good!
However we have the same issue with the background here. 
Can we do the same as we do with the orange background in the onboarding intro? There it looks good in fullscreen desktop.

**Comment by louilinn** [comment]
Screenshot from 2022-01-05 14-30-48
Screenshot from 2022-01-05 14-42-41

File: src/views/OnboardingOrganization.js (hunk 1 above)
Comment (mikozet) [reply src/views/OnboardingOrganization.js:254]: yes exactly, any atribute without value by default is 'true'

**Comment by mikozet** [comment]
> I think code looks good! However we have the same issue with the background here. Can we do the same as we do with the orange background in the onboarding intro? There it looks good in fullscreen desktop.

I corrected it now:
Zrzut ekranu 2022-01-5 o 16 07 12

**Comment by mikozet** [comment]
More adjustments taking into consideration todays talk with Lucia:

Zrzut ekranu 2022-01-7 o 15 42 50
Zrzut ekranu 2022-01-7 o 15 42 36
Zrzut ekranu 2022-01-7 o 15 40 21
Zrzut ekranu 2022-01-7 o 15 40 05
Zrzut ekranu 2022-01-7 o 15 39 58

Desktop looks a bit strange - we can add additional breakpoint but maybe also ask Lucia how she sees it..

**Comment by mikozet** [comment]
> Desktop looks a bit strange - we can add additional breakpoint but maybe also ask Lucia how she sees it..

We decided to solve it as it is in Onboarding tutorial

**Comment by louilinn** [comment]
Moved to https://github.com/CirclesUBI/circles-myxogastria/pull/307

**Pull request closed without merging**